code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -444,10 +444,11 @@
 	damage_types = list(BRUTE = 60)
 	embed = TRUE
 	armor_penetration = 60
-	agony = 70
+	agony = 100
 	penetrating = 2
 	affective_damage_range = 9
 	affective_ap_range = 9
+	penetrating = -5
```

Comment:
Why are there two vars named **exactly** the same with different values...? This bullet type already had `penetrating = 2` and now has an extra `penetrating = -5`

Reply:
I didnt see the penetrating 2